docs/renderer.md

```diff
@@ -0,0 +1,61 @@
+---
+name: Renderer
+route: '/renderer'
+---
+
+# Render Engine
+
+## Overview
+
+React requires a rendering engine, typically when creating an application people use `react-dom`.
```

Comment:
Perhaps a mention of `react-native` using `react-test-renderer` as the defacto renderer for tests (at least I think it is)?